option optimize_for = LITE_RUNTIME;

message BuyWudiFightBuffAnswer
{
 required uint32 result = 1;
 required uint32 index = 2;
 required uint32 boss_ts = 3;
 required uint32 buff_index = 4;
}
